pkg/api/vlabs/validate.go

```diff
@@ -32,6 +47,15 @@ func (o *OrchestratorProfile) Validate() error {
 	return nil
 }
 
+// ValidateOrchestratorVersion validates orchestrator version
+func (o *OrchestratorProfile) ValidateOrchestratorVersion(orchestratorVersion OrchestratorVersion) error {
```

Comment:
Do we validate version against existing versions? E.g. if orchestratortype is Kubernetes, then only 1.6.2, 1.6.0, and 1.5.3 is supported?